test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -6486,9 +6486,6 @@ def test_embedding_bag(self):
         offset = torch.tensor([0, 2, 5, 6])
         self.run_test(model, (input, offset))
 
-    @disableScriptTest()  # error in propagate as assign input shape
```

Comment:
Can you please update the PR description with more details?
In particular, I can see why constant folding is an improvement (we should do it whenever possible), but I would imagine any ops that we don't support for constant folding should not cause failures. Is there still a bug with handling shape propagation of `ReduceProd` nodes? Will we trigger that failure if we disable constant folding when exporting? If not, why not?

Reply:
appended with a line of comment in description.

> Is there still a bug with handling shape propagation of ReduceProd nodes?

The "shape" that gets used later on in this use case, is the value data in ReduceProd nodes. So normal shape propagation wouldn't help. We need to compute ReduceProd, and store the value in `ConstantValueMap::ShapeValue`, for further "symbolic" shape inference later.

> Will we trigger that failure if we disable constant folding when exporting?

No, it is symbolic shape inference that utilizes part of the constant folding code to compute some shape value. It is unrelated to the `do_constant_folding` feature that folds model parameters.

Reply:
Can you add these details to the PR description?